service/src/lib.rs

```diff
@@ -39,59 +35,50 @@ pub trait Factory: Send + Sync {
     fn get_metadata(&self) -> DeploymentMetadata;
 }
 
-/// Used to get resources of type `T` from factories.
-///
-/// This is mainly meant for consumption by our code generator and should generally not be called by users.
-///
-/// ## Creating your own managed resource
-///
-/// You may want to create your own managed resource by implementing this trait for some builder `B` to construct resource `T`.
-/// [`Factory`] can be used to provision resources on Shuttle's servers if your service will need any.
-///
-/// Please refer to `shuttle-examples/custom-resource` for examples of how to create custom resource. For more advanced provisioning
-/// of custom resources, please [get in touch](https://discord.gg/shuttle) and detail your use case. We'll be interested to see what you
-/// want to provision and how to do it on your behalf on the fly.
-///
-/// ```
```

Comment:
Why are we removing the steps that document how the community can create their own plugins?

Reply:
Yeah I should add it back but not as "your own *managed* resource"